### DISCUSSION TITLE
<input> does not focus in Popover

### DISCUSSION THREAD

**Discussion by iakovalgaev** [tailwindlabs/headlessui#1024] [Help]
In the Dialog, <input> works fine. But as soon as I'm going to place it inside the Popover, when I click it, it doesn't focus. I think it's related to events, but I spent two hours on it without solving the problem.

**Comment by keygun-development** [comment]
You can use `focus="true"` on the popoverpanel for it to automatically focus that when clicking the popoverbutton.
Example:
```
<Popover>
    <PopoverButton>
     click me
    </PopoverButton>

    <PopoverPanel focus>
        <input type="text" name="search" id="search" placeholder="input content">
    </PopoverPanel>
</Popover>
 ``` 

See https://headlessui.com/react/popover#popover-panel -> focus
